packages/admin-next/dashboard/src/components/layout/notifications/notifications.tsx

```diff
@@ -31,10 +52,111 @@ export const Notifications = () => {
+            HttpTypes.AdminNotificationListResponse,
+            HttpTypes.AdminNotification,
+            HttpTypes.AdminNotificationListParams
+          >
+            responseKey="notifications"
+            queryKey={["notifications"]}
+            queryFn={(params) => sdk.admin.notification.list(params)}
+            queryOptions={{ enabled: open }}
+            renderItem={(notification) => {
+              return (
+                <Notification
+                  key={notification.id}
+                  notification={notification}
+                />
+              )
+            }}
+          />
+        </Drawer.Body>
       </Drawer.Content>
     </Drawer>
   )
 }
+
+const Notification = ({
+  notification,
+}: {
+  notification: HttpTypes.AdminNotification
+}) => {
+  const data = notification.data as unknown as NotificationData | undefined
+
+  // We need at least the title to render a notification in the feed
+  if (!data?.title) {
+    return null
+  }
+
+  return (
+    <>
+      <div className="align-center mt-2 flex flex-row justify-center px-5">
+        <div className="text-ui-fg-muted mr-3">
+          <InformationCircleSolid />
+        </div>
+        <div className="w-full">
+          {/* We set the line height to 1 so the icon and text are aligned */}
+          <div className="align-center mb-1 flex flex-row justify-between">
+            <Label weight="plus" className="leading-4">
+              {data.title}
+            </Label>
+            <Text
+              as={"span"}
+              className="text-ui-fg-subtle leading-4"
+              size="small"
+            >
+              {formatDistance(notification.created_at, new Date(), {
+                addSuffix: true,
+              })}
+            </Text>
+          </div>
+
+          {!!data.description && (
+            <Text
+              className="text-ui-fg-subtle whitespace-pre-line"
+              size="small"
+            >
+              {data.description}
+            </Text>
+          )}
+          <NotificationFile file={data.file} />
+        </div>
+      </div>
+
+      <Divider className="my-4" />
+    </>
+  )
+}
+
+const NotificationFile = ({ file }: { file: NotificationData["file"] }) => {
+  if (!file?.url) {
+    return null
+  }
+
+  return (
+    <div className="my-3 flex flex-col">
+      <div className="shadow-elevation-card-rest bg-ui-bg-component transition-fg rounded-md px-4 py-2">
+        <div className="flex items-center gap-4">
+          <div className="flex w-full flex-row items-center justify-between">
+            <Text size="small" leading="compact">
+              {file?.filename ?? file.url}
+            </Text>
+            <IconButton variant="transparent" className="ml-2">
+              <a href={file.url} download={file.filename ?? `${Date.now()}`}>
+                <ArrowDownTray />
+              </a>
+            </IconButton>
+          </div>
+        </div>
+      </div>
+    </div>
+  )
+}
```

Comment:
The CSS is a bit off compared to Figma

```suggestion
const NotificationFile = ({ file }: { file: NotificationData["file"] }) => {
  if (!file?.url) {
    return null
  }

  return (
    <div className="shadow-elevation-card-rest bg-ui-bg-component transition-fg rounded-md px-3 py-2">
      <div className="flex w-full flex-row items-center justify-between gap-2">
        <Text size="small" leading="compact">
          {file?.filename ?? file.url}
        </Text>
        <IconButton variant="transparent" asChild>
          <a href={file.url} download={file.filename ?? `${Date.now()}`}>
            <ArrowDownTray />
          </a>
        </IconButton>
      </div>
    </div>
  )
}
```